[BUG] `BaggingForecaster` fails `test_update_with_exogenous_variables`

Tests have highlighted that `BaggingForecaster` fails some of the suite tests.

The failure is as below, to reproduce run `check_estimator` on an up-to-date environment.

```
FAILED sktime/forecasting/tests/test_all_forecasters.py::TestAllForecasters::test_update_with_exogenous_variables[BaggingForecaster-y:2cols-update_params=False] - ValueError: (time) indices of series 0 are not contained in index of series 0, extra indices are: DatetimeIndex(['2020-01-01', '2020-01-02', '2020-01-03', '2020-01-04',
               '2020-01-05', '2020-01-06', '2020-01-07', '2020-01-08',
               '2020-01-09', '2020-01-10', '2020-01-11', '2020-01-12',
               '2020-01-13', '2020-01-14', '2020-01-15', '2020-01-16',
               '2020-01-17', '2020-01-18', '2020-01-19', '2020-01-20'],
              dtype='datetime64[ns]', freq='D')
```

--- a/sktime/forecasting/compose/_bagging.py
+++ b/sktime/forecasting/compose/_bagging.py
@@ -97,9 +97,6 @@ class BaggingForecaster(BaseForecaster):
         "capability:pred_int:insample": True,  # ... for in-sample horizons?
         "capability:random_state": True,
         "property:randomness": "derandomized",
-        # CI and test flags
-        # -----------------
-        "tests:skip_by_name": ["test_update_with_exogenous_variables"],  # bug 8832
     }
 
     def __init__(

